accloudtant/aws/reports.py

```diff
@@ -119,6 +156,12 @@ def print_report(self):
                 instance.best,
             ]
             table.append(row)
+        if len(table) == 0:
+            row = [os.environ['AWS_DEFAULT_REGION'],'NONE','NONE','NONE','NONE','NONE','NONE','NONE','NONE','NONE','NONE']
```

Comment:
This might use `get`, to avoid problems when the environment variable is not defined.